### PR TITLE
Added interest model and controller. controller has 2 GET methods, ge…

### PR DESCRIPTION
**Pull request by amillion3**
Added interest model and controller. controller has 2 GET methods, getAllFavorites() and getSingleFavorite()

### CODE REVIEW

File: ClinkedIn/Models/InterestModel.cs [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+using Microsoft.AspNetCore.Mvc;
+
+namespace ClinkedIn.Models
+{
+    public class InterestModel
+    {
+        public InterestType Type { get; set; }
+    }
+
+    public enum InterestType
```
Comment (ghost) [inline ClinkedIn/Models/InterestModel.cs]: Can we put the InterestType inside of the InterestModel class?

Comment (amillion3) [reply ClinkedIn/Models/InterestModel.cs]: Certainly seems we can! Done, I'll push up that change in a minute.

**Comment by BLRussell-09** [comment]
Looking good

**Comment by BLRussell-09** [comment]
do it

**Pull request closed without merging**